interface/patient_file/summary/pnotes_full_add.php

```diff
@@ -14,7 +14,6 @@
 require_once("$srcdir/pnotes.inc");
 require_once("$srcdir/patient.inc");
 require_once("$srcdir/acl.inc");
-require_once("$srcdir/log.inc");
 require_once("$srcdir/options.inc.php");
 require_once("$srcdir/gprelations.inc.php");
 
```

Comment:
won't this need the following in the use section:
```
use OpenEMR\Common\Logging\EventAuditLogger;
```

Reply:
> Something to consider with use statements is if a globals require is not in the same scope as the use statement then chances are the class is out of scope as well.

I'm not sure which way you'd want to go on this. In the end, the autoloader is the only require needed - even for globals if altered accordingly - but that's a long way off. Is it safe to require_once the globals everywhere? Or would a direct autoloader reference be ok?